pkgs/development/python-modules/OWSLib/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, dateutil
+, pep8
+, pillow
+, pyproj
+, pytz
+, pytest
+, pytestcov
+, requests
+, tox
+}:
+
+buildPythonPackage rec {
+  pname = "OWSLib";
+  version = "0.16.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0m05225g1sqd2i8r2riaan33953hfni9wjq7n225snhl7klsb5gc";
+  };
+
+  buildInputs = [ dateutil pep8 pillow pyproj pytz requests tox ];
+  checkInputs = [ pytest pytestcov ];
+
+  doCheck = false;
```

Comment:
Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.

Reply:
No tests run and the build fails because no test run. I figured this was a bug in the python build infrastructure.